docs/SDK/Explanation/authentication/methods.md

```diff
@@ -0,0 +1,39 @@
+# Authentication Methods
+
+There are various authentication methods to obtain an AuthSig:
```

Comment:
wondering if we could better differentiate between an authsig for a pkp vs. an authsig used to obtain session sigs for a pkp.

at the moment, it seems like we're warning them against using authsigs and then saying to generate authsigs for session sigs.